lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1817,6 +1787,28 @@ void LinearMapInfo::populateLinearMapStructDeclarationFields(
       }
     }
   }
+
+  // Print generated linear map structs and branching trace enums.
+  // These declarations do not show up with `-emit-sil` because they are
+  // implicit. Instead, using `-Xllvm -debug-only=differentiation` to test
```

Comment:
```suggestion
  // implicit. Instead, use `-Xllvm -debug-only=differentiation` to test
```

Will fix in next patch.

Reply:
Done in https://github.com/apple/swift/pull/27269.